data/mods/Magiclysm/Spells/stormshaper.json

```
@@ -430,5 +431,461 @@
+    "id": "shocking_teleport",
+    "name": "Shocking Teleport",
+    "description": "Fill your legs with energy, and dash into picked place, ignoring all obstacles in between.",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "NO_HANDS", "LOUD", "TARGET_TELEPORT" ],
+    "effect": "short_range_teleport",
+    "shape": "blast",
+    "min_range": 4,
+    "max_range": 20,
+    "range_increment": 0.7,
+    "spell_class": "STORMSHAPER",
+    "base_casting_time": 500,
+    "final_casting_time": 100,
+    "casting_time_increment": -16,
+    "base_energy_cost": 550,
+    "final_energy_cost": 300,
+    "energy_increment": -10,
+    "max_level": 25,
+    "energy_source": "MANA"
+  },
+  {
+    "id": "shocking_teleport_damage",
+    "type": "SPELL",
+    "name": { "str": "Shocking Teleport damage" },
+    "description": "Deal some damage around the teleportation.  You can see it only in debug mode.",
+    "//": "extra_effects for shocking_teleport, currently doesn't work, because of various issues in dash/teleport and extra_effects incompabilies.",
+    "valid_targets": [ "ally", "hostile", "ground" ],
+    "effect": "attack",
+    "effect_str": "zapped",
+    "shape": "blast",
+    "max_level": 25,
+    "min_damage": 10,
+    "max_damage": 85,
+    "damage_increment": 3,
+    "damage_type": "electric",
+    "flags": [ "LOUD" ],
+    "min_duration": 100,
+    "max_duration": 500,
+    "min_aoe": 1,
+    "max_aoe": 7,
+    "aoe_increment": 0.25,
+    "field_id": "fd_electricity",
+    "field_chance": 1,
+    "min_field_intensity": 1,
+    "max_field_intensity": 10,
+    "field_intensity_increment": 0.4,
+    "field_intensity_variance": 3
+  },
+  {
+    "id": "toxtricity",
+    "type": "SPELL",
+    "name": "Toxtricity",
+    "description": "Inject an electric energy into the creatures, which makes them weak, and deal huge damage over time.",
+    "effect": "attack",
+    "effect_str": "zapped",
+    "shape": "blast",
+    "valid_targets": [ "self", "ally", "hostile", "ground" ],
+    "flags": [ "SOMATIC", "VERBAL", "NO_LEGS" ],
+    "max_level": 25,
+    "min_damage": 0,
+    "max_damage": 13,
+    "damage_increment": 0.5,
+    "min_aoe": 1,
+    "max_aoe": 11,
+    "aoe_increment": 0.4,
+    "min_range": 6,
+    "max_range": 21,
+    "range_increment": 0.6,
+    "min_duration": 0,
+    "max_duration": 1500,
+    "duration_increment": 60,
+    "base_casting_time": 350,
+    "final_casting_time": 150,
+    "casting_time_increment": -8,
+    "base_energy_cost": 400,
+    "final_energy_cost": 125,
+    "energy_increment": -11,
+    "energy_source": "MANA",
+    "difficulty": 5,
+    "spell_class": "STORMSHAPER",
+    "min_dot": 0,
+    "max_dot": 15,
+    "dot_increment": 0.6,
+    "damage_type": "electric"
+  },
+  {
+    "id": "electric_eye",
+    "type": "SPELL",
+    "name": "Electric Eye",
+    "description": "Enhance your sight with your powers.  It allows you to see creatures of the same energy around you, and some minor bonuses.",
```

```suggestion
    "description": "Enhance your sight with your powers.  It allows you to see creatures of the same energy around you. Enhanced perception speed enables you to see attacks coming, allowing for an additional block or dodge attempt.",
```
"Other minor bonuses" is not useful information.